src/com/theironyard/novauc/CustomMath.java

```diff
@@ -0,0 +1,16 @@
+package com.theironyard.novauc;
+
+import javafx.scene.shape.Circle;
+import javafx.scene.shape.TriangleMesh;
+
+/**
+ * Created by Eric on 2/7/17.
+ */
+public class CustomMath {
+
+    Circle o = new Circle(1.5);
+
+
+    TriangleMesh t = new TriangleMesh(a +b = c)
+
```

Comment:
Good job Eric.  The requirements were met well and I have no critical feedback on those.  I like that you gave hardmode a go.  All you really did was call constructors to make new objects.  
This is more what we were looking for
 public double areaCircle(double radius) {
        double area = Math.PI * (Math.pow(radius, 2));
        return area;
    }

    public double hypotenuseFinder(double a, double b) {
        double hypotenuse = Math.sqrt((Math.pow(a, 2)) + (Math.pow(b, 2)));
        return hypotenuse;
    }

if you have questions about how those work let me know.  Again really nicely done